src/odemis/gui/cont/tabs.py

```diff
@@ -1882,7 +1883,7 @@ def _on_add_file(self) -> None:
     def _on_add_tileset(self) -> None:
         self.select_acq_file(extend=True, tileset=True)
 
-    def load_tileset(self, filenames: list[str], extend: bool = False) -> None:
+    def load_tileset(self, filenames: List, extend: bool = False) -> None:
```

Comment:
You can still write `List[str]`. You can actually just write `list`. Just not `list[str]`.